runtime/vm/callin.cpp

```diff
@@ -366,7 +366,7 @@ buildCallInStackFrame(J9VMThread *currentThread, J9VMEntryLocalStorage *newELS,
 	return success;
 }
 
-static void
+void
```

Comment:
Since this function is decorated w/ `#if JAVA_SPEC_VERSION >= 19` at `vm_api.h`, `static` can stay for pre-JDK19+ levels.